web6/web6.go

```diff
@@ -801,3 +801,21 @@ func dynamicPage_404(uri string, w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusNotFound)
 	w.Write([]byte(base_html))
 }
+
+func dynamicPage_500(error_msg string, w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "text/html")
+
+	//TODO(z): Create a formatted 500 Internal Server Error html page like dynamicPage_404 and return that instead of text
+	error_response := fmt.Sprintf("500 Internal Error. %v", error_msg)
+
+	w.WriteHeader(http.StatusInternalServerError)
+	w.Write([]byte(error_response))
+}
+
+func recoverError_500(w http.ResponseWriter, r *http.Request) {
+	// Recover from internal panics until we could guarantee no panics
+	if recover := recover(); recover != nil {
+		error_message := fmt.Sprintf("Internal panic: %v", recover)
```

Comment:
If we're going to return the stack to the user we should have this wrapped in a config option (in a prod setup we wouldn't want this to return to the user -- we'd probably just want a 5xx page)

Reply:
Will comment the stack message for now - if we need it we'll add it + the config options